google-cloud-spanner/lib/google/cloud/spanner/batch_snapshot.rb

```diff
@@ -173,6 +173,14 @@ def timestamp
         #   * `:optimizer_version` (String) The version of optimizer to use.
         #     Empty to use database default. "latest" to use the latest
         #     available optimizer version.
+        # @param [Hash] call_options A hash of values to specify the custom
+        #   call options, e.g., timeout, retries, etc. Call options are
+        #   optional. The following settings can be provided:
+        #
+        #   * `:timeout` (Numeric) A numeric value of custom timeout in seconds
+        #     that overrides the default setting.
+        #   * `:retry_policy` (Hash) A hash of values that overrides the default
```

Comment:
Can you add attributes docs for retry_policy or ref to doc link.

i.e https://github.com/googleapis/google-cloud-ruby/blob/8fc158a134ea29a8435f6681b20f9e2600d3e0b7/google-cloud-spanner-v1/lib/google/cloud/spanner/v1/spanner/client.rb#L1761-L1767

Reply:
Thanks for the suggestion. Fixed.